src/System.Windows.Forms/src/System/Windows/Forms/ListViewItem.ListViewSubItem.ListViewSubItemAccessibleObject.cs

```diff
@@ -25,16 +24,22 @@ public ListViewSubItemAccessibleObject(ListViewSubItem? owningSubItem, ListViewI
                 {
                     OwningSubItem = owningSubItem;
                     _owningItem = owningItem.OrThrowIfNull();
-                    _owningListView = owningItem.ListView ?? owningItem.Group?.ListView ?? throw new InvalidOperationException(nameof(owningItem.ListView));
                 }
 
-                internal override UiaCore.IRawElementProviderFragmentRoot FragmentRoot
-                    => _owningListView.AccessibilityObject;
+                private ListView? owningListView => _owningItem.ListView ?? _owningItem.Group?.ListView;
```

Comment:
Capitalize "O" because it's property
```suggestion
                private ListView? OwningListView => _owningItem.ListView ?? _owningItem.Group?.ListView;
```